app/controllers/dashboard_controller.rb

```diff
@@ -1,7 +1,10 @@
 class DashboardController < ApplicationController
   before_action :require_current_user
 
-  def index; end
+  def index
+    @user = current_user
+    @friends = @user.followed
+  end
```

Comment:
I think we could try and refactor using a facade here further down the line

Reply:
Thanks! Thats a good suggestion. Yeah possibly do a facade, or perhaps a callback for the user